xenit-alfresco/templates/acs/acs-config.yaml

```
@@ -45,7 +42,16 @@ data:
   GLOBAL_transform.service.enabled: 'false'
   {{- end }}
   GLOBAL_aos.baseUrlOverwrite: '{{ .Values.ingress.syncServiceHost }}://{{ .Values.ingress.host }}/alfresco/aos'
-  {{- if not .Values.solr.enabled }}
+  {{- if .Values.solr.enabled }}
+  GLOBAL_index.subsystem.name: "solr6"
+  GLOBAL_solr.port: '30300'
+  {{- else if .Values.elastic.enabled }}
+  GLOBAL_index.subsystem.name: "elasticsearch"
+  GLOBAL_elasticsearch.host: {{ .Values.elastic.host | quote }}
+  GLOBAL_elasticsearch.port: {{ .Values.elastic.port | quote }}
+  GLOBAL_elasticsearch.indexName: {{ (((.Values.global).elasticsearch).indexName) | default (printf "alfresco-%s" .Release.Name) | quote }}
```

```suggestion
  GLOBAL_elasticsearch.indexName: {{ .Values.global.elasticsearch.indexName | default (printf "alfresco-%s" .Release.Name) | quote }}
```

These brackets have no meaning

---

Strangely they do, @thijslemmens  was also surprised about this.
it stops evaluating the chain if it hits null, otherwise it crashes with a NIL exception.